src-static/update_adm0_sf.R

```diff
@@ -157,63 +156,63 @@ st_crop_adj_bbox <- function(sf_obj, xmin = 0, xmax = 0, ymin = 0, ymax = 0) {
 
 #' Download ADM0 shapefile
 #'
-#' Takes in an `iso3` code, and returns the country shapefile for that country.
+#' Takes in an `iso3` code, and returns the location shapefile for that location
 #'
 #' For some `iso3` codes, custom files are required, and these are handled directly
 #' within this function. However, for the rest, the default is to try and pull
-#' the COD shapefile from Fieldmaps (because it has standardized columns and files
-#' for easy access). If CODs are unavailable, then the country shape is pulled
+#' the COD shapefile from FieldMaps (because it has standardized columns and files
+#' for easy access). If CODs are unavailable, then the location shape is pulled
 #' from the UN Geodata downloaded from the geodata portal.
 #'
 #' https://geoportal.un.org/arcgis/home/item.html?id=d7caaff3ef4b4f7c82689b7c4694ad92
 #'
 #' @param iso3 ISO3
-#' @param update_azure `logical` if TRUE (default) update json file in azure
-#'     if FALSE
-download_adm0_sf <- function(iso3, update_azure = TRUE) {
+download_adm0_sf <- function(iso3) {
   if (iso3 == "LAC") {
     # for LAC we get all 3 of El Salvador, Guatemala, and Honduras
     dplyr$filter(sf_world, iso3cd %in% c("SLV", "GTM", "HND")) |>
       dplyr$group_by(iso3cd) |>
       # pull together polygons by iso3cd
       dplyr$summarise(do_union = TRUE, .groups = "drop") |>
       # then merge all iso3cds into 1 multipolygon retaining boundaries
-      dplyr$summarise(do_union = FALSE)
+      dplyr$summarise(boundary_source = "UN Geo Hub", do_union = FALSE)
 
   } else if (iso3 == "AB9") {
-    download_shapefile(
+    download_shapefile$download_shapefile(
       url = "https://open.africa/dataset/56d1d233-0298-4b6a-8397-d0233a1509aa/resource/76c698c9-e282-4d00-9202-42bcd908535b/download/ssd_admbnda_abyei_imwg_nbs_20180401.zip", # nolint
       layer = "ssd_admbnda_abyei_imwg_nbs_20180401",
-      data_source  = "Open Africa"
+      boundary_source  = "Open Africa"
     )
   } else if (iso3 == "XKX") {
-    download_shapefile(
+    download_shapefile$download_shapefile(
       url = "https://data.geocode.earth/wof/dist/shapefile/whosonfirst-data-admin-xk-latest.zip",
       layer = "whosonfirst-data-admin-xk-country-polygon",
-      data_soure = "Who's On First"
+      boundary_source = "Who's On First"
     )
   } else if (iso3 == "IOT") {
-    download_shapefile(
+    download_shapefile$download_shapefile(
       url = "https://public.opendatasoft.com/api/explore/v2.1/catalog/datasets/world-administrative-boundaries/exports/geojson?lang=en&timezone=Europe%2FLondon",#nolint
-      data_source = "Opendatasoft"
+      boundary_source = "Opendatasoft"
     ) |> # nolint
```

Comment:
what will the lintr not like here?

Reply:
Dunno, wasn't added in this PR, I imagine maybe was kept when the `data_source` was added maybe if the line was previously one long line?

Reply:
Have removed!